channeldb/kvdb/etcd/fixture_test.go

```diff
@@ -103,9 +109,9 @@ func (f *EtcdTestFixture) Dump() map[string]string {
 	ctx, cancel := context.WithTimeout(context.TODO(), testEtcdTimeout)
 	defer cancel()
 
-	resp, err := f.cli.Get(ctx, "", clientv3.WithPrefix())
+	resp, err := f.cli.Get(ctx, "\x00", clientv3.WithFromKey())
```

Comment:
Rationale for this change?

Reply:
Or will a `nil` work here instead?

Reply:
Seems like this is the standard way to do this, see https://github.com/etcd-io/etcd/issues/9833#issuecomment-399177511
Unfortunately the old way of empty string plus `WithPrefix()` didn't work as etcd complained that the key is empty.